ragstack/colbert/cassandra_store.py

```diff
@@ -129,7 +129,7 @@ def __create_tables(self) -> None:
         self.session.execute(
             f"""
             CREATE CUSTOM INDEX IF NOT EXISTS colbert_ann_{self.table_name} ON {self.full_table_name}(bert_embedding) USING 'StorageAttachedIndex'
-  WITH OPTIONS = {{'similarity_function': 'DOT_PRODUCT' }};
+  WITH OPTIONS = {{'source_model': 'bert' }};
```

Comment:
we probably will need an option to specify if it is cassandra or astra.  and use this change only for astra.